draft-ietf-gnap-core-protocol.md

```diff
@@ -953,6 +1044,9 @@ string-type resource items.
     ]
 ~~~
 
+The requested access is the sum of all elements of the array, including both objects and 
```

Comment:
I think "union" is a better word than "sum".

Reply:
yes you're right